src/javascript/binary/common_functions/valid_account_opening.js

```diff
@@ -57,7 +57,7 @@ var ValidAccountOpening = (function(){
       errorFname.innerHTML = Content.errorMessage('min', '2');
       Validate.displayErrorMessage(errorFname);
       window.accountErrorCounter++;
-    } else if (!/^[a-zA-Z\s-.']+$/.test(fname.value)){
+    } else if (/[`~!@#$%^&*)(_=+\[}{\]\\\/";:\?><,|\d]+/.test(fname.value)){
```

Comment:
why is this regexp different to the one that Sasha used in the JSON schema, i.e. "^[\\p{L}\\s'.-]{2,30}$" ?